### DISCUSSION TITLE
alchemy problem

### DISCUSSION THREAD

**Discussion by chainflash** [Cyfrin/foundry-full-course-cu#2875] [General]
![Screenshot (26)](https://github.com/user-attachments/assets/de864f2b-13eb-48aa-9fc6-855768544e2a)

I have a problem, when I want to use alchemy node and I enter the information correctly, I get this problem.
Can anyone help?

Error summary :
Error: HTTP error 403 with body: <!DOCTYPE html>

**Comment by EngrPips** [comment]
Hello @chainflash, Please try get another `rpc-url` then retry.

**Reply by chainflash** [reply]
The problem was with my IP, it was fixed thanks

**Reply by EngrPips** [reply]
Awesome, I am glad you figured it out. Keep crushing it.
